app/assets/javascripts/short-form/ShortFormDataService.js.coffee

```diff
@@ -393,30 +392,30 @@ ShortFormDataService = (ListingService) ->
     preferences = Service._initPreferences(data)
     shortFormPrefs = angular.copy(sfApp.shortFormPreferences) || []
     shortFormPrefs.forEach( (shortFormPref) ->
-
-      listingPref = ListingService.getPreferenceById(shortFormPref.listingPreferenceID)
+      listing = sfApp || ListingDataService.listing
```

Comment:
what's sfAPP here?

Reply:
my guess is that it's an application object, in a format that is recently retrieved from salesforce and hasn't been through web app-specific formatting yet. not 100% sure. but this line is a typo, it should be `listing = sfApp.listing || ListingDataService.listing`, i'll update that